src/st/np3/bss.c

```diff
@@ -1,4 +1,4 @@
-#include "common.h"
+#include <game.h>
```

Comment:
It's not worth pulling this out into a header here, but part of the header refactor PR was to avoid signatures like this where previously a few dozen lines of type defs were being included, and now 5K+ lines of type defs, global refs, and all kinds of unrelated things are now part of compiling this file.

I don't think it's a blocker and this file with eventually go away anyway.

Reply:
Yeah I am not a fan of this either. But unfortunately there's so much we can do by going into hypothesis on how the game was originally structured. It's like the current C files with weird names: we will slowly find a meaningful organisation for them.